app/models/health_status.php

```diff
@@ -260,7 +261,7 @@ function ($entity) use ($path) {
     $stats = Health::getStats($projects);
     $translated = $stats['translated'];
     $reference = $stats['total'];
-    $completion = round(($translated / $reference) * 100, 2);
+    $completion = $reference === 0 ? 0 : round(($translated / $reference) * 100, 2);
```

Comment:
ternary assignation is cryptic to me there :) Mind adding a comment explaining what you are doing? Thanks